content/root/quick-start-guide.textile

```diff
@@ -353,6 +380,15 @@ channel
     }
   );
 
+bc[go]. channel := client.Channels.Get("quickstart")
+_, err = channel.Subscribe(context.Background(), "greeting", func(msg *ably.Message) {
+  fmt.Printf("Received message : '%v'\n", msg.Data)
+})
+if err != nil {
+  err := fmt.Errorf("Error subscribing to channel: %w", err)
+  fmt.Println(err)
```

Comment:
I think we should be consistent with error handling across the examples, so either panic with the error:

```go
if err != nil {
	panic(err)
}
```

or assume the examples are within an outer function with a single `error` return value, and so just return the error:

```go
if err != nil {
	return fmt.Errorf("error subscribing to channel: %w", err)
}
```

Reply:
Updated all samples with `panic(err)`, thanks.